### PR TITLE
Add osd-utils-cli to Openshift CI

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

### CODE REVIEW

**Comment by yeya24** [comment]
Hello @cblecker, please take a look when you are free, thanks!

**Comment by cblecker** [comment]
/lgtm

/assign @hongkailiu
for approval

File: ci-operator/jobs/openshift/osd-utils-cli/OWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+approvers:
+- clcollins
+- rogbas
+- fahlmant
+options: {}
+approvers:
```
Comment (rogbas) [inline ci-operator/jobs/openshift/osd-utils-cli/OWNERS:6]: We have `approvers:` twice, shouldn't it reviewers?

Comment (yeya24) [reply ci-operator/jobs/openshift/osd-utils-cli/OWNERS:8]: Updated. Please take a look again

**Comment by cblecker** [comment]
/assign @stevekuznetsov @alvaroaleman 

Are either of you able to approve for a new repos CI? Thank you!

**Pull request merged**